src/frontend/e2e/entityPopup.spec.ts

```diff
@@ -0,0 +1,68 @@
+import { test, expect } from '@playwright/test';
+
+test('entity test', async ({ page }) => {
+  await page.goto('/');
+  await page.getByRole('button', { name: 'Sign in' }).click();
+  await page
+    .getByLabel('', { exact: true })
+    .locator('div')
+    .filter({ hasText: "Temporary AccountIf you're" })
+    .nth(3)
+    .click();
+
+  // click first project card on the home page
+  await page.locator('.MuiCardContent-root').first().click();
+
+  // click on task & assert task popup visibility
+  await page.waitForTimeout(4000);
+  await page.locator('canvas').click({
+    position: {
+      x: 388,
+      y: 220,
+    },
+  });
+  await expect(page.getByText('Status: READY')).toBeVisible();
+  await expect(page.getByRole('button', { name: 'START MAPPING' })).toBeVisible();
+
+  // click on entity within task & assert feature popup visibility
+  await page.waitForTimeout(4000);
+  await page.locator('canvas').click({
+    position: {
+      x: 387,
+      y: 211,
+    },
+  });
+  await expect(page.getByRole('heading', { name: 'Feature:' })).toBeVisible();
+  await expect(page.getByRole('button', { name: 'MAP FEATURE IN ODK' })).toBeEnabled();
+  await page.getByRole('button', { name: 'MAP FEATURE IN ODK' }).click();
```

Comment:
After this click I think we should expect a console warning that the redirect to protocol `odkcollect://` failed (it attempts to open the link via mobile intent).

We could set a console listener via playwright and check this warning is displayed (it's not a user facing thing, but it helps check the button works as intended)